playwright/tests/pageobjects/CodeEditor.ts

```diff
@@ -36,6 +38,16 @@ class CodeEditor {
     await expect(this.code).toBeVisible();
   }
 
+  async openBrowsers() {
+    await this.focus();
+    await this.scriptArea.getByLabel('Browser').click();
+    await expect(this.page.waitForSelector('.browser-content')).toBeDefined();

```

Comment:
I don't think we have ever worked with toBeDefined but mostly with toBeVisible(). But I'm not sure if this is a big difference. @ivy-lli is there a best practice for this?